website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -36,9 +36,19 @@ export default {
       task: {},
     };
   },
+  generateURL (message) {
```

Comment:
this needs to be placed inside `methods: { }`, or else it won't be recognized as a vue method